standards/github_folder_template.md

```diff
@@ -0,0 +1,50 @@
+# The .github folder and it's functionality
+
+### What is the .github folder?
+
+The `.github` folder allows users to store GitHub related files/folders. Some of these files and folders perform special functions for organizing your repo, issues, and PRs.
+
+Here are some of those functions:
+- Creating issue templates ([ISSUE_TEMPLATE folder](../.github/ISSUE_TEMPLATE))
+- Creating PR templates 
+- Creating a Code of Conduct
+- Gathering Funding for a project
+- Settings/config files for GitHub Actions
+- Security (reporting security vulnerabilities)
+- Code Owners (set default PR reviewers)
+
+Here is how the `.github` folder looks in this repo.
+![github folder](./assets/github_folder/1.png)
+
+### How to create an issue template
+Creating issue templates allows you to save time when creating issues. They are also beneficial for making sure assignees are including adequate information in the PR. In order to create an issue template, you must 
```

Comment:
What do you think about showing the end result of having image templates so the developers will understand how they save time? In #258 there is an example image that showcases our `Issue`, `Bug`, and `Epic` templates as they would be displayed to the developer. I think that would make a good example image here.